src/day1/expense_report_fix.rb

```diff
@@ -0,0 +1,83 @@
+#! /usr/bin/env -S ruby -w
+
+#: AoC Day1 solutions. (See `problems.txt` for the exact problems
+#: definitions.)
+
+require 'optparse'
+require 'ostruct'
+
+require_relative '../utils'
+require_relative './array_ext'
+
+module ExpenseReportFix
+
+  def ExpenseReportFix.scan_pairs(amounts, expected_sum = 2020)
```

Comment:
When defining module-level methods the idiom is to do `extend self` in the beginning.
https://www.bigbinary.com/blog/extend-self-in-ruby

Reply:
Thanks, `extend self` is a nice thing! (I don't think I've seen it in any of the books, btw). This one fixed in 5c87212; also reworked `ScanPasswds` and `AocUtils` the same way.